spec/index.bs

```diff
@@ -1186,36 +1186,94 @@ partial interface ShadowRoot {
 
 ### Enforcement for scripts ### {#enforcement-in-scripts}
 
+This document modifies how {{HTMLScriptElement}} [=child text content=] can be set to allow applications to control dynamically created scripts. It does so by
+modifying the {{HTMLElement/innerText}} and {{Node/textContent}} attributes. The behavior of the attributes remains the same
+as in their original counterparts, apart from additional behavior similar to that triggered by the {{StringContext}} extended attribute.
+
+Note: Using these IDL attributes is the recommended way of dynamically setting the URL or a text of a script. Manipulating attribute nodes or text nodes directly will call a default policy on the final value when the script is prepared.
+
+<pre class="idl exclude">
+partial interface Node {
+  [CEReactions] attribute (DOMString or TrustedScript)? textContent;
+};
```

Comment:
This seems extremely ungainly (same for `innerText`). Given that there's a fallback, can we just rely on the fallback and only make `.text` magic?

Reply:
Unfortunately we probably can't as it wouldn't be compatible with existing code.

Currently setting textContent or innerText to a trusted type would work, if we made this change it would be treated like any other string and require the default policy. Which sites might not even implement.

Reply:
Hmm, maybe we could shadow these members on `script` elements, but I'm not sure we should pollute the entire node tree with this stuff.

cc @smaug----

Reply:
Yeah if we could find a way to only touch the script element IDL that'd be ideal.

Reply:
We still need to cover for the case when the setters from the `Node` are used on scripts, no? Otherwise that's a bypass.

Reply:
Not sure how this could work on script element only. One could still use the setter from Node interface, no?
@petervanderbeken

Reply:
So how is setting `textNode.data` protected? Or any of the other ways to change the child nodes of a `script` element?

Reply:
The script element has an internal slot which won't match it's text content and so will go through the default policy.

Reply:
So why would that not happen when `Node`'s setter is used? Seems like that would be fine.

Reply:
That would happen but if you don't have a default policy that's not web compatible. Because currently something that works will stop working.

Reply:
Sure, but it's somewhat involved to actually do that, so the likelihood of that happening is small.

Reply:
> so the likelihood of that happening is small

Okay yeah fair, to be more accurate, it's a web compatibility risk.

I'm personally okay with this as a change if it makes the spec and implementation easier, and it's probably fine compat wise?

I'll update this PR to do it that way instead.

Reply:
Taking a look into how the IDL will look for this and then I'll push a commit up here that should hopefully make both the integration AND the spec much simpler at the cost of some potential edge cases if people's existing code is doing wonky stuff.